BM/instruction-check/feature_list.py

```diff
@@ -145,6 +145,14 @@
         "cpuid": ['7', '0', '1', '0', 'a', '7'],
         "platforms": {"SRF", "CWF"}
     },
+    "lam": {
```

Comment:
All the feature name in this file are capital letter.

Reply:
updated with capital letter for feature name.